rust/operator-binary/src/main.rs

```diff
@@ -77,6 +82,13 @@ async fn main() -> anyhow::Result<()> {
                 &cluster_info_opts,
             )
             .await?;
+            let event_recorder = Arc::new(Recorder::new(
```

Comment:
suggestion: Leave a small note here explaining why we need the `Reported` and the `Arc`.

Reply:
Pls have a look at 8aacd70